### DISCUSSION TITLE
Problem with request of GitHub education

### DISCUSSION THREAD

**Discussion by pabuce** [community/community#46171] [GitHub Education] [answered]
Hello,

I have problems to request my github education plan, because when I upload mi id card photo, the page tells me:

Your application cannot be reviewed until you fix the following:
The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.,You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

I would like to recieve some help.

Thanks

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @pabuce 

I found 2 answered discussions about this topic, maybe it can help you:

#20819
#44112
